web/templates/web/index.html

```diff
@@ -36,7 +36,7 @@
     <div class="make-content-padding">{# Empty div for spacing #}</div>
 
     <div class="ui container">
-        {% include 'news/event/index_event_list_contents.html' with event_dicts=featured_event_dicts ribbon_extra_classes="yellow make-bg-yellow" %}
+        {% include 'news/event/index_event_list_contents.html' with event_dicts=featured_event_dicts ribbon_extra_classes="yellow make-bg-yellow" secondary_ribbon="blue make-bg-blue" %}
```

Comment:
See my comment on `event_list.html` about the ribbon classes variable names 🙂

Reply:
I now changed this